lib/App/Sqitch/Engine.pm

```diff
@@ -269,8 +257,17 @@ sub deploy {
     $self->$meth( $plan, $to_index );
 }
 
-sub revert {
-    my ( $self, $to ) = @_;
+sub revert($$$$) {
+    # $to = revert up to (but not including) this change. May be undefined.
+    # $prompt = If true, we ask for confirmation; if false, we don't.
+    # $prompt_default = Default if the user just hits enter at the prompt.
+    my ( $self, $to, $prompt, $prompt_default ) = @_;
+
+    if (defined $prompt) {
+        hurl revert => 'Missing mandatory parameter $prompt_default' unless defined $prompt_default;
```

Comment:
These should be internationalized by using, IIRC,

``` perl
 hurl revert => __x(
    'Missing mandatory parameter {param}',
    param => '$prompt_default',
) unless defined $prompt_default;
```

Reply:
Agreed, though $prompt_default is not a variable here, we want to print a literal `$`.